rotkehlchen/chain/evm/transactions.py

```diff
@@ -458,7 +459,8 @@ def address_has_been_spammed(self, address: ChecksumEvmAddress) -> bool:
             ):
                 for tx_hash in erc20_tx_hashes:
                     raw_receipt_data = self.evm_inquirer.get_transaction_receipt(tx_hash)
-                    for log_entry in raw_receipt_data['logs']:
+                    detected_transfers = 0
+                    for log_pos, log_entry in enumerate(raw_receipt_data['logs']):
```

Comment:
We almost always call this `idx`. I was trying to figure out what `log_pos` was for a bit. Would have called it also maybe `log_idx` but that can be something else too

```suggestion
                    for idx, log_entry in enumerate(raw_receipt_data['logs']):
```